### PR TITLE
Improvements to MNE Scan, mne_rt_server, README.md and disp3D

### PR DESCRIPTION
**Pull request by LorenzE**
- Change logo in README.md to point to SVG in repository
- Refactored MNE Scan Neuromag client plugin
- Refactored mne_rt_server Neuromag connector plugin
- Fix short data type parsing in Fiff library
- Rename/Add static building flag to examples and applications
- Update OpenGL version testing in Disp3D data model
- Rename DoxyGen class descriptions in header files 
- Fix block size editing field in FiffSimulator and Neuromag plugins in MNE Scan
- Remove Applications and examples building from windows appveyor test job

### CODE REVIEW

File: applications/mne_rt_server/connectors/Neuromag/dacqserver.h [hunk 1]
```diff
@@ -42,49 +41,44 @@
...
-
-#include <QByteArray>
+#include <QSharedPointer>
+#include <QPointer>
 
 
 //*************************************************************************************************************
 //=============================================================================================================
-// DEFINE NAMESPACE NeuromagPlugin
+// FORWARD DECLARATIONS
```
Comment (chdinh) [inline applications/mne_rt_server/connectors/Neuromag/dacqserver.h]: Isn't it the namespace definition?

File: applications/mne_rt_server/connectors/Neuromag/dacqserver.h [hunk 2]
```diff
@@ -42,49 +41,44 @@
...
 
-using namespace FIFFLIB;
+namespace NEUROMAGRTSERVERPLUGIN
+{
 
 
 //*************************************************************************************************************
 //=============================================================================================================
-// FORWARD DECLARATIONS
+// NEUROMAGRTSERVERPLUGIN FORWARD DECLARATIONS
```
Comment (chdinh) [inline applications/mne_rt_server/connectors/Neuromag/dacqserver.h]: Why not just keeping FORWARD DECLARATIONS

File: applications/mne_rt_server/mne_rt_server/commandthread.h [hunk 3]
```diff
@@ -29,7 +29,7 @@
 * POSSIBILITY OF SUCH DAMAGE.
 *
 *
-* @brief     implementation of the CommandThread Class.
+* @brief     Definition of the CommandThread Class.
```
Comment (chdinh) [inline applications/mne_rt_server/mne_rt_server/commandthread.h]: Must be Declaration

File: applications/mne_rt_server/mne_rt_server/connectormanager.cpp [hunk 4]
```diff
@@ -29,7 +29,7 @@
 * POSSIBILITY OF SUCH DAMAGE.
 *
 *
-* @brief     implementation of the ConnectorManager Class.
+* @brief     Definition of the ConnectorManager Class.
```
Comment (chdinh) [inline applications/mne_rt_server/mne_rt_server/connectormanager.cpp]: Must be Declaration

File: applications/mne_rt_server/mne_rt_server/connectormanager.h [hunk 5]
```diff
@@ -29,7 +29,7 @@
 * POSSIBILITY OF SUCH DAMAGE.
 *
 *
-* @brief     implementation of the ConnectorManager Class.
+* @brief     Definition of the ConnectorManager Class.
```
Comment (chdinh) [inline applications/mne_rt_server/mne_rt_server/connectormanager.h]: Must be Declaration

File: applications/mne_rt_server/mne_rt_server/fiffstreamserver.h [hunk 6]
```diff
@@ -29,7 +29,7 @@
 * POSSIBILITY OF SUCH DAMAGE.
 *
 *
-* @brief     implementation of the FiffStreamServer Class.
+* @brief     Definition of the FiffStreamServer Class.
```
Comment (chdinh) [inline applications/mne_rt_server/mne_rt_server/fiffstreamserver.h]: Must be Declaration

File: applications/mne_rt_server/mne_rt_server/fiffstreamthread.h [hunk 7]
```diff
@@ -29,7 +29,7 @@
 * POSSIBILITY OF SUCH DAMAGE.
 *
 *
-* @brief     implementation of the FiffStreamThread Class.
+* @brief     Definition of the FiffStreamThread Class.
```
Comment (chdinh) [inline applications/mne_rt_server/mne_rt_server/fiffstreamthread.h]: Must be Declaration

File: applications/mne_rt_server/mne_rt_server/mne_rt_server.h [hunk 8]
```diff
@@ -29,7 +29,7 @@
 * POSSIBILITY OF SUCH DAMAGE.
 *
 *
-* @brief     implementation of the MNERTServer Class.
+* @brief     Definition of the MNERTServer Class.
```
Comment (chdinh) [inline applications/mne_rt_server/mne_rt_server/mne_rt_server.h]: Must be Declaration

File: tools/design/logos/MNE-CPP_Logo.svg [hunk 9]
```diff
@@ -1,8 +1,8 @@
 <?xml version="1.0" encoding="utf-8"?>
```
Comment (chdinh) [inline tools/design/logos/MNE-CPP_Logo.svg]: Did the Logo Change? - Hard to review if this is okay?

File: applications/mne_rt_server/connectors/Neuromag/dacqserver.h (hunk 1 above)
Comment (LorenzE) [reply applications/mne_rt_server/connectors/Neuromag/dacqserver.h]: No these are the forward declarations. The namespace declarations follows below.

File: applications/mne_rt_server/connectors/Neuromag/dacqserver.h (hunk 2 above)
Comment (LorenzE) [reply applications/mne_rt_server/connectors/Neuromag/dacqserver.h]: These are the forward declarations inside of the NEUROMAGRTSERVERPLUGIN namespace. The global forward declarations above are specified as FORWARD DECLARATIONS

File: applications/mne_rt_server/mne_rt_server/connectormanager.cpp (hunk 4 above)
Comment (LorenzE) [reply applications/mne_rt_server/mne_rt_server/connectormanager.cpp]: actually this is the source file so I think Definition should be fine

File: tools/design/logos/MNE-CPP_Logo.svg (hunk 9 above)
Comment (LorenzE) [reply tools/design/logos/MNE-CPP_Logo.svg]: Nope. I did not change the logo. I did open it with Illustrator though. I think this should be fine.

**Pull request merged**